src/api/objects/ObjectAPISpec.js

```diff
@@ -248,10 +248,17 @@ describe('The Object API', () => {
       });
 
       it('displays a notification in the event of an error', () => {
-        mockProvider.get.and.returnValue(Promise.reject());
+        openmct.notifications.warn = jasmine.createSpy('warn');
+        mockProvider.get.and.returnValue(
+          Promise.reject({
+            name: 'Error',
+            status: 404,
+            statusText: 'Not Found'
+          })
+        );
 
         return objectAPI.get(mockDomainObject.identifier).catch(() => {
-          expect(openmct.notifications.error).toHaveBeenCalledWith(
+          expect(openmct.notifications.warn).toHaveBeenCalledWith(
```

Comment:
huh. why did this change? was this test a false positive before?

Reply:
Yeah, for record, I'll update what we talked about offline, this was passing because of the object interceptor's notifications.error call. so now we're testing object API specifically.